sdk/iOS/OoyalaSkinSDK/React/OOReactSkinBridgeModuleMain.m

```diff
@@ -146,6 +146,16 @@ - (void)setSkinViewDelegate:(nonnull id<OOReactSkinModelDelegate>)delegate {
   }
 }
 
+RCT_EXPORT_METHOD(onSwitch:(NSDictionary *)params) {
+  NSMutableDictionary *result = [[NSMutableDictionary alloc] initWithDictionary:params];
```

Comment:
Sorry, I’ve missed this part 
Why do we need to create a separate dictionary?
It’s safe to access properties directly in `params` dictionary

Reply:
Done, the dictionary was removed.